core/domain/summary_services.py

```diff
@@ -187,7 +201,7 @@ def _generate_query(categories):
         summary_dicts = [
             all_summary_dicts[exp_id] for exp_id in exp_ids_to_display
             if exp_id in all_summary_dicts]
-
+        summary_dicts = sorted(summary_dicts, key=lambda k: k['created_on'])
```

Comment:
Wait, why are we arranging all the summary_dicts for each group this way? I thought the intention was to add a new group to the top of the library, not modify the existing ones.